x-pack/plugins/index_management/__jest__/client_integration/helpers/test_subjects.ts

```diff
@@ -9,6 +9,9 @@ export type TestSubjects =
   | 'aliasesTabContent'
   | 'appTitle'
   | 'cell'
+  | 'indexManagementHome div div:nth-child(2) div:nth-child(2)'
```

Comment:
Is this necessary?

Reply:
Sorry. Wasn't supposed to ping you yet. The tests aren't quite complete.